### PR TITLE
Colormath (PyPI)

### PR DESCRIPTION
**Pull request by ewels**
Package for [colormath](https://github.com/gtaylor/python-colormath) - a python module that abstracts common color math operations. For example, converting from CIE L*a*b to XYZ, or from RGB to CMYK

### CODE REVIEW

File: recipes/colormath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (synapticarbors) [inline recipes/colormath/meta.yaml]: Drop blank line

File: recipes/colormath/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - networkx
```
Comment (synapticarbors) [inline recipes/colormath/meta.yaml]: `networkx` is not required to build. please remove.

File: recipes/colormath/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+about:
+  home: https://github.com/gtaylor/python-colormath
+  summary: 'Color math and conversion library.'
+  description: |
+    This module implements a large number of different
+    color operations such as color space conversions,
+    Delta E, and density to spectral.
+  license: BSD
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/colormath/meta.yaml]: Add `license_file: LICENSE.txt`

Comment (synapticarbors) [inline recipes/colormath/meta.yaml]: add `dev_url: https://github.com/gtaylor/python-colormath` and `doc_url: http://python-colormath.readthedocs.io/en/latest/`

Comment (ewels) [inline recipes/colormath/meta.yaml]: Done 👍

Comment (ewels) [inline recipes/colormath/meta.yaml]: Done 👍

File: recipes/colormath/meta.yaml (hunk 1 above)
Comment (ewels) [inline recipes/colormath/meta.yaml]: As in my other PR, GitHub is being confusing about which line this comment is on. Head of file or between `{% set` and `package`?

Comment (synapticarbors) [inline recipes/colormath/meta.yaml]: head of the file

Comment (ewels) [inline recipes/colormath/meta.yaml]: ok, in that case also done :) 👍

Comment (ewels) [inline recipes/colormath/meta.yaml]: If you're curious, this is why I was confused:

image


image

**Comment by ewels** [comment]
Test failure seems to just be a request timeout..

**Comment by ewels** [comment]
To confirm - if the [failing job](https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.13720/job/i4rwq76buyf4i91n) can be restarted then I think all jobs will pass. The error message is as follows:

```
Runtime error: HTTPError: 504 Server Error: Gateway Time-out for url: https://conda.anaconda.org/conda-forge/win-64/filelock-2.0.6-py35_0.tar.bz2
```

**Comment by ewels** [comment]
Hi @synapticarbors,

Let me know if you need anything else doing. I'm a bit anxious to get this merged as it's needed for the [spectra package](https://github.com/conda-forge/staged-recipes/pull/2989) which is itself needed for my [MultiQC](http://multiqc.info/) bioconda package. I released a major new version of this last week and people keep asking me when they can get it through conda :)

Cheers,

Phil

**Comment by synapticarbors** [comment]
@ewels Sorry for the delay, I was at a conference and review a lot of recipes so sometimes it takes me some time to swing back around to one I previously looked at. 

My final comment is that in looking at the `setup.py` of the package, it does not appear as if `numpy` is actually required to build the recipe, and since no C-extensions are being built based on the Numpy C-API, I think you can drop `numpy x.x` in favor of just `numpy` in the `run` section. 

Otherwise everything else looks good to me.

**Comment by ewels** [comment]
Ok great, thanks! No problems with the delay at all - I appreciate your help! Numpy changes pushed.

Phil

**Comment by ewels** [comment]
Thanks! :tada:

**Comment by ewels** [comment]
Sorry @synapticarbors - am I just being too impatient again or has something gone wrong with the automated packaging? The [colormath-feedstock](https://github.com/conda-forge/colormath-feedstock) repo has been created but it's empty, and I haven't had any e-mails about being added to a team for it or anything..

**Comment by ewels** [comment]
Never mind, it came though. Thanks again!

**Pull request merged**